packages/vite/src/shared/hmr.ts

```diff
@@ -28,7 +28,7 @@ export interface HMRConnection {
   /**
    * Send message to the client.
    */
-  send(messages: string): void
+  send(messages: HotPayload): void
```

Comment:
It is also typed in the Environment API documentation

Reply:
Thanks, docs updated.